myhpi/templates/footer.html

```diff
@@ -6,7 +6,7 @@
     {% for column in footer_columns %}
     <div class="footer-category">
       {% with column|markdown as parsed_markdown %}
-      {{ parsed_markdown.0|tag_external_links }}
+      {{ parsed_markdown.0 }}
```

Comment:
doesnt't that mean that you could just send the body of a page through the same old filter and the issue would be resolved?

Reply:
True that. I'll take the old filter then (soon™)

Reply:
Very fair argument, although i wonder to what extend i should apply this filter (alias, where should i put it apart from the content block?)

Right now, that very filter is also marking local links in the sidebar
![image](https://github.com/fsr-de/myHPI/assets/21260178/e5d1cce3-580f-44e3-9dd4-63be4e721f35)
But i guess that's because the filter checks for `settings.SITE_URL`, so it shouldn't be a problem in production, right? Still, this feels quite sketchy imo.

Reply:
I don't think that you should apply it to ToC, as this only renders headings and headings should never be hyperlinks (apart from the ToC where you navigate to the anchor). But if you only apply it to the main content it should be fine I think?

Reply:
Sadly, it also shows internal links in the main content as external. Probably, this works in production but probably won't in local setups...

Reply:
yeah I don't think that this can be fixed apart from setting the SITE_URL